package.json

```diff
@@ -4,41 +4,41 @@
     "sortier": "./bin/index.js"
   },
   "dependencies": {
-    "@types/postcss-less": "4.0.0",
-    "@typescript-eslint/typescript-estree": "4.24.0",
-    "angular-html-parser": "1.8.0",
-    "cosmiconfig": "7.0.0",
-    "find-up": "5.0.0",
-    "flow-parser": "0.151.0",
-    "globby": "11.0.3",
-    "ignore": "5.1.8",
-    "postcss-less": "4.0.1",
-    "postcss-scss": "3.0.5",
-    "typescript": "4.2.4"
+    "@types/postcss-less": "^4.0.0",
+    "@typescript-eslint/typescript-estree": "^4.24.0",
+    "angular-html-parser": "^1.8.0",
+    "cosmiconfig": "^7.0.0",
+    "find-up": "^5.0.0",
+    "flow-parser": "^0.151.0",
+    "globby": "^11.0.3",
+    "ignore": "^5.1.8",
+    "postcss-less": "^4.0.1",
+    "postcss-scss": "^3.0.5",
+    "typescript": "^4.2.4"
   },
   "description": "An opionated code sorter",
   "devDependencies": {
-    "@snowcoders/renovate-config": "2.0.3",
-    "@types/estree": "0.0.47",
-    "@types/globby": "8.0.0",
-    "@types/jest": "26.0.23",
-    "@types/node": "15.3.1",
-    "@typescript-eslint/eslint-plugin": "4.24.0",
-    "@typescript-eslint/parser": "4.24.0",
-    "changelog-updater": "2.0.2",
-    "coveralls": "3.1.0",
-    "eslint": "7.26.0",
-    "eslint-config-prettier": "8.3.0",
-    "eslint-plugin-prettier": "3.4.0",
-    "husky": "5.1.3",
-    "jest": "26.6.3",
-    "lint-staged": "11.0.0",
-    "prettier": "2.3.0",
-    "rimraf": "3.0.2",
-    "source-map-support": "0.5.19",
-    "ts-jest": "26.5.6",
-    "ts-node": "9.1.1",
-    "tslint": "6.1.3"
+    "@snowcoders/renovate-config": "^2.0.3",
```

Comment:
Pin all dev dependencies, these aren't shipped so the less unstable the better. See https://docs.renovatebot.com/dependency-pinning/#so-whats-best for details of when to pin and when not to pin